tmt/utils.py

```diff
@@ -852,34 +852,42 @@ def format(
     return output
 
 
-def create_directory(path, name, quiet=False):
+def create_directory(path, name, dry=False, quiet=False):
     """ Create a new directory, handle errors """
     say = log.debug if quiet else echo
     if os.path.isdir(path):
         say("Directory '{}' already exists.".format(path))
         return
     try:
-        os.makedirs(path, exist_ok=True)
-        say("Directory '{}' created.".format(path))
+        if dry:
```

Comment:
In order to prevent unnecessary nesting let's move the `if dry` part above `try` and call `return` at the end of the block.